### PR TITLE
[fix](Outfile) Fix the data type mapping for complex types in Doris to the ORC and Parquet file formats.

### PR DESCRIPTION
**Pull request by BePPPower**
### What problem does this PR solve?

Problem Summary:

As before, the behavior of exporting of complex data types in Doris is as follows:
  | orc type | parquet type | csv
-- | -- | -- | --
bitmap | string | Not Supported | Not Supported
quantile_state | Not Supported | Not Supported | Not Supported
hll | string | string | invisible string
jsonb | Not Supported | string | string
variant | Not Supported | string | string

What's more, there are some issues when exporting complex data types to the ORC file format.

This PR does two things:
1. Fix the problem with exporting complex data types from Doris.
2. Support exporting these three complex types to both the ORC and the Parquet file format.

  | orc type | parquet type | csv
-- | -- | -- | --
bitmap | binary | binary | "NULL"
quantile_state | binary | binary | "NULL"
hll | binary | binary | "NULL"
jsonb | string | string | string
variant | string | string | string

### Release note

None

### Check List (For Author)

- Test 
    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [x] Yes. 

- Does this need documentation?
    - [ ] No.
    - [x] Yes. https://github.com/apache/doris-website/pull/1370

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

File: be/src/vec/data_types/serde/data_type_hll_serde.cpp [hunk 1]
```diff
@@ -48,28 +49,17 @@ Status DataTypeHLLSerDe::serialize_column_to_json(const IColumn& column, int64_t
 Status DataTypeHLLSerDe::serialize_one_cell_to_json(const IColumn& column, int64_t row_num,
                                                     BufferWritable& bw,
                                                     FormatOptions& options) const {
-    if (!options._output_object_data) {
```
Comment (morningman) [inline be/src/vec/data_types/serde/data_type_hll_serde.cpp:51]: only export use this option `_output_object_data`?

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run buildall

**Comment by BePPPower** [comment]
run p0

**Comment by BePPPower** [comment]
run p0

**Comment by BePPPower** [comment]
run buildall

**Review by morningman — APPROVED** [review]

**Review by CalvinKirs — APPROVED** [review]

**Pull request merged**